lib/scrabble.rb

```diff
@@ -1,9 +1,52 @@
 class Scrabble
 
   def score(word)
-    1
+    if word == nil || word == ""
+      return 0
+    end
+    letter_arr = word.upcase.chars
+    score_total = 0
+    letter_arr.each do |letter|
```

Comment:
when you are using each to modify an external variable, theres a better variable to use